src/rhsm/connection.py

```diff
@@ -897,24 +899,31 @@ def _print_debug_info_about_request(
                     msg += blue_col + f" {tunel_headers}" + end_col
                 msg += "\n"
 
-            if self.insecure is True:
-                msg += blue_col + f"Making insecure ({auth}) request:" + end_col
-            else:
-                msg += blue_col + f"Making ({auth}) request:" + end_col
+            msg += green_col + "Request:\n" + end_col
             msg += (
                 red_col
-                + " https://"
+                + "https://"
                 + f"{normalized_host(self.host)}:{safe_int(self.ssl_port)}{handler} {request_type}"
```

Comment:
If you are significantly changing debug printing, then please print `request_type` first to get something like this:

```
Request:
GET https://centos8-candlepin:8443/candlepin/ (using consumer auth)
```